fuel-tx/src/tests/offset.rs

```diff
@@ -530,6 +530,46 @@ fn tx_offset_upload() {
     assert!(cases.output_contract_created_id);
 }
 
+#[test]
+fn tx_offset_blob() {
+    let mut cases = TestedFields::default();
+    let number_cases = 100;
+
+    // The seed will define how the transaction factory will generate a new transaction.
+    // Different seeds might implicate on how many of the cases we cover - since we
+    // assert coverage for all scenarios with the boolean variables above, we need to
+    // pick a seed that, with low number of cases, will cover everything.
+    TransactionFactory::<_, Upload>::from_seed(1295)
+        .take(number_cases)
+        .for_each(|(tx, _)| {
+            let bytes = tx.to_bytes();
+            chargeable_transaction_parts(&tx, &bytes, &mut cases);
```

Comment:
It would be nice to check getters for `blob_id`.

Reply:
4fe7322e